cluster/src/main/java/com/hartwig/pipeline/tertiary/virusbreakend/VirusBreakend.java

```diff
@@ -0,0 +1,104 @@
+package com.hartwig.pipeline.tertiary.virusbreakend;
+
+import java.util.List;
+
+import com.hartwig.pipeline.Arguments;
+import com.hartwig.pipeline.ResultsDirectory;
+import com.hartwig.pipeline.alignment.AlignmentPair;
+import com.hartwig.pipeline.datatypes.DataType;
+import com.hartwig.pipeline.execution.PipelineStatus;
+import com.hartwig.pipeline.execution.vm.BashCommand;
+import com.hartwig.pipeline.execution.vm.BashStartupScript;
+import com.hartwig.pipeline.execution.vm.VirtualMachineJobDefinition;
+import com.hartwig.pipeline.metadata.AddDatatype;
+import com.hartwig.pipeline.metadata.ArchivePath;
+import com.hartwig.pipeline.metadata.SomaticRunMetadata;
+import com.hartwig.pipeline.report.EntireOutputComponent;
+import com.hartwig.pipeline.report.Folder;
+import com.hartwig.pipeline.report.RunLogComponent;
+import com.hartwig.pipeline.reruns.PersistedDataset;
+import com.hartwig.pipeline.reruns.PersistedLocations;
+import com.hartwig.pipeline.resource.ResourceFiles;
+import com.hartwig.pipeline.storage.GoogleStorageLocation;
+import com.hartwig.pipeline.storage.RuntimeBucket;
+import com.hartwig.pipeline.tertiary.TertiaryStage;
+
+public class VirusBreakend extends TertiaryStage<VirusBreakendOutput> {
+
+    public static final String NAMESPACE = "virusbreakend";
+
+    private final ResourceFiles resourceFiles;
+    private final PersistedDataset persistedDataset;
+
+    public VirusBreakend(final AlignmentPair alignmentPair, final ResourceFiles resourceFiles, final PersistedDataset persistedDataset) {
+        super(alignmentPair);
+        this.resourceFiles = resourceFiles;
+        this.persistedDataset = persistedDataset;
+    }
+
+    @Override
+    public String namespace() {
+        return NAMESPACE;
+    }
+
+    @Override
+    public List<BashCommand> commands(final SomaticRunMetadata metadata) {
+        return List.of(
+                // TODO implement class to export PATH of latest version of tool
+                () -> "export PATH=/opt/tools/gridss/2.11.1:$PATH",
```

Comment:
we already have a handy class actually, have a look at ExportPathCommand

Reply:
Sweet stuff, using it now ;)